### DISCUSSION TITLE
Why it didn't work for me

### DISCUSSION THREAD

**Discussion by suntong** [goreleaser/goreleaser#2146] [Q&A] [answered]
I forked `goreleaser/example` and made some commits

https://github.com/suntong/example/commits/master

Why it is not working for me? I was never able to trigger the build and see the releases. 

What I'm missing? 

PS. I don't have docker so I remove all docker related, but I don't think that's the cause of the problem. 

Somebody help please. Thanks

**Comment by caarlos0** [comment, marked as answer]
I think github actions don't run on forks... you may need to "unfork" it (aka delete the fork, create a new repo with the same name and push your changes).

**Reply by suntong** [reply]
BINGO! Thanks!
